src/odemis/gui/cont/tabs.py

```diff
@@ -3310,21 +3315,17 @@ def __init__(self, name, button, panel, main_frame, main_data):
         else:
             self.panel.btn_manualfocus.Bind(wx.EVT_BUTTON, self._onManualFocus)
 
-        if "center-align" in tab_data.align_mode.choices:
+        if "center-align" not in tab_data.align_mode.choices:
```

Comment:
I'd recommend to inverse the `if`, because it is a little easier to read when the condition is not negated. (so move the negative into the else block).

Reply:
Done ! I did it that way because the condition about "lens-align" was done by checking the negative first. Maybe we could change this one to be clearer.